JhipsterBlazor/Pages/Account/Register.razor.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Blazored.Modal.Services;
+using JhipsterBlazor.Models;
+using JhipsterBlazor.Services.AccountServices;
+using JhipsterBlazor.Shared.Constants;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+using Newtonsoft.Json.Linq;
+
+namespace JhipsterBlazor.Pages.Account
+{
+    public partial class Register : ComponentBase
+    {
+        [Inject]
+        private IRegisterService RegisterService{ get; set; }
+
+        [Inject]
+        private IModalService ModalService { get; set; }
+
+        private RegisterModel RegisterModel = new RegisterModel();
+
+        private bool Success { get; set; }
+        private bool Error { get; set; }
+        private bool DoNotMatch { get; set; }
+        private bool ErrorEmailExists { get; set; }
+        private bool ErrorUserExists { get; set; }
+
+
+
+        protected override async Task OnInitializedAsync()
+        {
```

Comment:
We don't need to call SetAllErrorFalse here ? for reset all errors when the user go back and return on registration view ?

Reply:
I just try this and the errors reset :)